kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverVersions.kt

```diff
@@ -21,4 +23,10 @@ public object KoverVersions {
      * JaCoCo coverage tool version used by default.
      */
     public const val JACOCO_TOOL_DEFAULT_VERSION = "0.8.10"
+
+    /**
+     * Current version of Kover Gradle Plugin
+     */
+    public val version: String
+        get() = KoverFeatures.getVersion()
```

Comment:
Are versions for Features artifact and Gradle plugin will always be the same?

Reply:
Yes